### PR TITLE
Java EE 8 convenience features

### PR DESCRIPTION
**Pull request by aguibert**

### CODE REVIEW

**Comment by aguibert** [comment]
#build

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature [hunk 1]
```diff
@@ -0,0 +1,28 @@
+-include= ~../cnf/resources/bnd/feature.props
```
Comment (ewhoch) [inline dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: Does this file have the right name? It says appserver.javaee-7.0.feature

Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: Isn't the name of this file incorrect?  It should be ...javaee-8.0.feature, correct?

Comment (aguibert) [reply dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: you're right -- need to fix that

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+IBM-App-ForceRestart: install, uninstall
+IBM-ShortName: javaee-8.0
+Subsystem-Version: 8.0.0
+Subsystem-Name: Java EE Full Platform 8.0
+# TODO-EE8 add batch-1.0 and ejb-3.2 once persistence service works w/ JPA 2.2
+-features=\
+  com.ibm.websphere.appserver.appClientSupport-1.0,\
+  com.ibm.websphere.appserver.concurrent-1.0,\
+  com.ibm.websphere.appserver.jacc-1.5,\
+  com.ibm.websphere.appserver.jaspic-1.1,\
```
Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: jaspic-1.1 is now part of Web Profile...

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  com.ibm.websphere.appserver.javaMail-1.6,\
+  com.ibm.websphere.appserver.jaxws-2.2,\
+  com.ibm.websphere.appserver.jca-1.7,\
+  com.ibm.websphere.appserver.jcaInboundSecurity-1.0,\
+  com.ibm.websphere.appserver.jms-2.0,\
+  com.ibm.websphere.appserver.j2eeManagement-1.1,\
+  com.ibm.websphere.appserver.servlet-4.0,\
```
Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: servlet is part of Web Profile...  Was servlet-3.2 in the full platform?  That was wrong, if that's the case.

File: dev/com.ibm.websphere.appserver.javaee-8.0/resources/l10n/com.ibm.websphere.appserver.javaee-8.0.properties [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature combines the Liberty features that support the Java EE 7.0 Full Platform.
```
Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.javaee-8.0/resources/l10n/com.ibm.websphere.appserver.javaee-8.0.properties]: Update description to indicate Java EE 8.0 Full Platform...

File: dev/com.ibm.websphere.appserver.webProfile-8.0/com.ibm.websphere.appserver.webProfile-8.0.feature [hunk 5]
```diff
@@ -0,0 +1,28 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.webProfile-8.0
+visibility=public
+IBM-App-ForceRestart: install, uninstall
+IBM-ShortName: webProfile-8.0
+Subsystem-Version: 8.0.0
+Subsystem-Name: Java EE Web Profile 8.0
+# TODO-EE8: Need to add appSecurity-3.0 once bug w/ CDI 2.0 is resolved
```
Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.webProfile-8.0/com.ibm.websphere.appserver.webProfile-8.0.feature]: appSecurity-3.0 will bring in jaspic-1.1, but we should probably list it explicitly...

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature (hunk 3 above)
Comment (aguibert) [reply dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-7.0.feature]: we need to specify servlet in the full profile feature in order to force a toleration chain from included features.  I checked and the same thing was done in javaee-8.0, so I don't think this is an error.

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-8.0.feature [hunk 6]
```diff
@@ -0,0 +1,27 @@
+-include= ~../cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.javaee-8.0
+visibility=public
+IBM-App-ForceRestart: install, uninstall
+IBM-ShortName: javaee-8.0
+Subsystem-Version: 8.0.0
+Subsystem-Name: Java EE Full Platform 8.0
+# TODO-EE8 add batch-1.0 and ejb-3.2 once persistence service works w/ JPA 2.2
```
Comment (ewhoch) [inline dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-8.0.feature]: Is there an issue open to hold onto this TODO? do we need it for sure or just a really nice to have?

File: dev/com.ibm.websphere.appserver.webProfile-8.0/resources/l10n/com.ibm.websphere.appserver.webProfile-8.0.properties [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=This feature combines the Liberty features that support the Java EE 8.0 Web Profile.
```
Comment (ewhoch) [inline dev/com.ibm.websphere.appserver.webProfile-8.0/resources/l10n/com.ibm.websphere.appserver.webProfile-8.0.properties]: Is it "Java EE Web Profile 8.0" or "Java EE 8.0 Web Profile" ? I suspect Kevin would have caught this if i was wrong.

Comment (aguibert) [reply dev/com.ibm.websphere.appserver.webProfile-8.0/resources/l10n/com.ibm.websphere.appserver.webProfile-8.0.properties]: this is a copy/pase of what we had for 7.0, with a version bump, so I think we should just keep in consistent

File: dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-8.0.feature (hunk 6 above)
Comment (aguibert) [reply dev/com.ibm.websphere.appserver.javaee-8.0/com.ibm.websphere.appserver.javaee-8.0.feature]: yes, this TODO is tracked by the blockers in the parent story to this PR (issue #811)

**Comment by aguibert** [comment]
#build

**Pull request merged**